darts/metrics/metrics.py

```diff
@@ -113,9 +114,27 @@ def wrapper_multivariate_support(*args, **kwargs):
     return wrapper_multivariate_support
 
 
+def _get_values(series: TimeSeries,
+                stochastic_quantile: Optional[float] = 0.5) -> np.ndarray:
+    """
+    Returns the numpy values of a time series.
+    For stochastic series, return either all sample values with (stochastic_quantile=None) or the quantile sample value
+    with (stochastic_quantile {>=0,<=1})
+    """
+    if series.is_deterministic:
+        series_values = series.univariate_values()
+    else:  # stochastic
+        if stochastic_quantile is None:
+            series_values = series.data_array().values
```

Comment:
You could call `series.all_values()` instead. And if you know what you're doing and that you won't be modifying the values, you can make it `series.all_values(copy=False)`, which will be faster.

Reply:
Nice, very good to know! Thanks 👍